cmd/http/main.go

```diff
@@ -26,7 +26,12 @@ import (
 func main() {
 	ctx := context.Background()
 
-	c, err := config.LoadConfig("/etc/config")
+	configPath := os.Getenv("CONFIG")
```

Comment:
@matthyx what do you think is the proper solution here? Because we are thinking of having multiple configuration files...

Reply:
Exactly, the goal is to split the config between component and cluster like in [node-agent](https://github.com/kubescape/node-agent/blob/main/pkg/config/config.go).